### PR TITLE
cloud: update pause operation behavior for TiDB Cloud Dedicated clusters

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/pause-or-resume-tidb-cluster.md
Comment (Oreoxmt) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:1]: Preview: https://pingcap-docsite-preview-git-preview-cl-bd7c1b-oreoxmts-projects.vercel.app/tidbcloud/pause-or-resume-tidb-cluster

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 1]
```diff
@@ -24,18 +24,51 @@ Comparing with backup and restore, pausing and resuming a cluster takes less tim
...
+> **Note:**
+>
+> If your organization was created before November 12, 2024, your cluster still follows the compatible pause behavior. TiDB Cloud will notify you before transitioning to the new standard pause behavior.
+
 When a cluster is paused, note the following:
 
 - TiDB Cloud stops collecting monitoring information of the cluster.
 - You cannot read data from or write data to the cluster.
 - You cannot import or back up data.
+- TiDB Cloud will not auto resume the paused cluster.
```
Comment (Oreoxmt) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:61]:
```suggestion
- TiDB Cloud does not automatically resume the paused cluster.
```

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 2]
```diff
@@ -24,18 +24,51 @@ Comparing with backup and restore, pausing and resuming a cluster takes less tim
 
 ## Pause a TiDB cluster
 
+The pause duration and behavior depend on your organization's creation date:
+
+- Organizations created after November 12, 2024, follow the standard pause behavior with a maximum pause duration of 7 days.
+- Organizations created before November 12, 2024, follow the compatible pause behavior, which allows a longer pause duration. These organizations will gradually transition to the standard 7-day limit.
```
Comment (lilin90) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:30]:
```suggestion
- Organizations created after November 12, 2024 follow the standard pause behavior with a maximum pause duration of 7 days.
- Organizations created before November 12, 2024 follow the compatible pause behavior, which allows a longer pause duration. These organizations will gradually transition to the standard 7-day limit.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 3]
```diff
@@ -24,18 +24,51 @@ Comparing with backup and restore, pausing and resuming a cluster takes less tim
 
 ## Pause a TiDB cluster
 
+The pause duration and behavior depend on your organization's creation date:
+
+- Organizations created after November 12, 2024 follow the standard pause behavior with a maximum pause duration of 7 days.
+- Organizations created before November 12, 2024 follow the compatible pause behavior, which allows a longer pause duration. These organizations will gradually transition to the standard 7-day limit.
```
Comment (Oreoxmt) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:30]:
```suggestion
- Organizations created on or before November 12, 2024 follow the compatible pause behavior, which allows a longer pause duration. These organizations will gradually transition to the standard 7-day limit.
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**